src/sentry/integrations/slack/actions/form.py

```diff
@@ -68,13 +68,14 @@ def clean(self) -> dict[str, Any] | None:
             channel_prefix = self.data["channel"][0] if self.data["channel"][0] == "@" else "#"
 
         cleaned_data: dict[str, Any] = super().clean()
+        assert cleaned_data is not None
```

Comment:
this type annotation is actually incorrect -- but I'd rather leave it such that when I introduce `django-stubs` this gets corrected.  `clean()` returns a `None | SomeDict` and then the narrowing here is correct